README.md

```diff
@@ -1 +1,98 @@
-# go-config
\ No newline at end of file
+# e-TF1 go-config
+
+Package e-TF1 go-config allows you to load your application configuration from multiple sources (dotenv, env, flags...).  
+This package is built on top of [confita](https://github.com/heetch/confita) 
+
+The main difference with confita is that the last loader that matches the config key will override the previous match.
```

Comment:
maybe add something like this

```suggestion
The main differences with confita are the following:
* the final config will aggregate all the loaders results.
* the last loader that matches the config key will override the previous match.
```

Reply:
but I'm still not sure that a phrase is clear enough compared to a demo or some code 👍